osbuild/testutil/__init__.py

```diff
@@ -155,3 +159,30 @@ def pull_oci_archive_container(archive_path, image_name):
         yield
     finally:
         subprocess.check_call(["skopeo", "delete", f"containers-storage:{image_name}"])
+
+
+def make_fake_service_fd() -> int:
+    """Create a file descriptor suitable as input for --service-fd for any
+    host.Service
+
+    Note that the service will take over the fd and take care of the
+    lifecycle so no need to close it.
+    """
+    sock = socket.socket(socket.AF_UNIX, socket.SOCK_SEQPACKET)
+    fd = os.dup(sock.fileno())
+    return fd
+
+
+def find_one_subclass_in_module(module: ModuleType, subclass: Type) -> object:
+    """Find the class in the given module that is a subclass of the given input
+
+    If multiple classes a found an error is raised.
```

Comment:
I think there is a typo here: "a" instead of "are" ?

Reply:
Nice catch, thank you! Fixed.